src/api/app/models/user.rb

```diff
@@ -64,6 +64,7 @@ class User < ApplicationRecord
 
   has_many :moderated_comments, class_name: 'Comment', foreign_key: 'moderator_id'
   has_many :decisions, foreign_key: 'moderator_id'
+  has_many :canned_responses
```

Comment:
We should destroy all canned responses of a user if the user gets deleted, so:

```suggestion
  has_many :canned_responses, dependent: :destroy
```

Reply:
The default strategy is to do nothing. This is what the documentation says in the link you shared:

> For has_many, (...) if no :dependent option is given, then it will follow the default strategy. The default strategy is to do nothing (leave the foreign keys with the parent ids set)